slidey_js.js

```diff
@@ -0,0 +1,42 @@
+var images = ["http://i.imgur.com/toiWjkX.jpg?1", "http://i.imgur.com/lvUZZ5h.jpg", "http://i.imgur.com/lN5Xj7Z.jpg", "http://i.imgur.com/Uiq53lr.jpg", "http://i.imgur.com/ogXWnCM.jpg", "http://i.imgur.com/QOgvT7o.jpg", "http://i.imgur.com/ODjDt.jpg", "http://i.imgur.com/0qJQV.jpg", "http://i.imgur.com/rlT9q.jpg"]
+var interval;
+var currentPic = 0;
+
+
+$(document).ready(function () {
+
+  var loopPics = function() {
+
+    interval = setInterval(setPics, 5000);
```

Comment:
Awesome solution! I feel like the pause functionality might not be working because your `interval` is a global variable, and is not being reassigned on line 35 here.